### ISSUE TITLE
CRS: 941310 test fails

### ISSUE THREAD

**Issue by bxlxx** [corazawaf/coraza#230]
The regex encoding format used by Golang is UTF-8. If we want to match non UTF-8 data, we need to convert the matching string into UTF-8 encoding format.

For example:
if we have the below code, the result of ```re.MatchString``` will be false.
```
func Test_rx(t *testing.T) {
	re, err := regexp.Compile(`\x8A`)
	if err != nil {
		t.Error(err)
		return
	}

	str := "\x8A"
	if re.MatchString(str) {
		t.Error("match error")
	}
}
```
We need to convert the str into the UTF-8 format of data, like this:
```
func Test_rx(t *testing.T) {
	re, err := regexp.Compile(`\x8A`)
	if err != nil {
		t.Error(err)
		return
	}

	str := "\x8A"
	if re.MatchString(convert(str)) {
		t.Error("match error")
	}
}

func convert(src string) string {
	var buf bytes.Buffer
	for i := range src {
		buf.WriteRune(rune(src[i]))
	}
	return buf.Bytes()
}
```

**Comment by jptosso** [comment]
Is this fixed with #231  ?

**Comment by bxlxx** [comment]
Replaced with #270
